avocado/plugins/spawners/remote.py

```diff
@@ -161,7 +161,8 @@ async def spawn_task(self, runtime_task):
                 )
                 return False
 
-        status, output = await RemoteSpawner.run_remote_cmd_async(session, shlex.join(entry_point_args), 14400)
+        cmd = shlex.join(entry_point_args) + " > /dev/null"
```

Comment:
This is an interesting observation, and while on other spawners it's not hogging aexpect, this is wasteful in most cases.

What if we default the `task-run` behavior so that if a status server is given, the standard outputs will not have duplicates of the messages?  There can be a flag that actually enables (back) that behavior if needed.

Reply:
> This is an interesting observation, and while on other spawners it's not hogging aexpect, this is wasteful in most cases.

I am actually not sure whether it could also be wasteful on other spawners since the loss in performance was quite significant (>20%) and the output ends up not being used anywhere (while it is fully captured in the logs).

> What if we default the `task-run` behavior so that if a status server is given, the standard outputs will not have duplicates of the messages? There can be a flag that actually enables (back) that behavior if needed.

I see, so you suggest there might be cases where this output might actually be needed? Do you have some concrete example of running tests without a status server and/or logging so that such output will only have to be captured from the remote stdout?

Reply:
> > What if we default the `task-run` behavior so that if a status server is given, the standard outputs will not have duplicates of the messages? There can be a flag that actually enables (back) that behavior if needed.
> 
> I see, so you suggest there might be cases where this output might actually be needed? Do you have some concrete example of running tests without a status server and/or logging so that such output will only have to be captured from the remote stdout?

No, I can not think of an example that the output is useful, other than when running `task-run` from the command line for debugging purposes.  But, it's quite easy to setup a server and look at its output instead.

I believe we reached an agreement of removing the wasteful output that `task-run` (and `runnable-run`) generate.